10.9-libcxx/stable/main/finkinfo/libs/pythonmods/cssselect-py.info

```diff
@@ -31,6 +36,10 @@ CompileScript: true
 InstallScript: <<
   python%type_raw[python] setup.py install --root=%d
 <<
+InfoTest: <<
+  TestDepends: pytest-py%type_pkg[python], lxml-py%type_pkg[python]
+  TestScript: %p/bin/python%type_raw[python] setup.py pytest || exit 2
```

Comment:
The TestScript is wrong:

```
/sw/bin/python3.7 setup.py pytest || exit 2
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'pytest'
```
I'd suggest just calling `%p/bin/pytest-%type_raw[python]` instead of running it through setup.py.

Reply:
Do we have different setuptools versions or something (here `setuptools-tng-py37-40.2.0-1`)?
```
/sw/bin/python3.7 setup.py pytest || exit 2
running pytest
running egg_info
creating cssselect.egg-info
writing cssselect.egg-info/PKG-INFO
writing dependency_links to cssselect.egg-info/dependency_links.txt
writing top-level names to cssselect.egg-info/top_level.txt
writing manifest file 'cssselect.egg-info/SOURCES.txt'
reading manifest file 'cssselect.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
no previously-included directories found matching 'docs/_build'
writing manifest file 'cssselect.egg-info/SOURCES.txt'
running build_ext
================================================================ test session starts ================================================================
platform darwin -- Python 3.7.1, pytest-3.6.4, py-1.5.4, pluggy-0.7.1
Matplotlib: 3.0.1
Freetype: 2.6.1
rootdir: /scratch.noindex/fink.build/cssselect-py37-1.0.3-1/cssselect-1.0.3, inifile: setup.cfg
plugins: mpl-0.10, mock-1.10.0, expect-1.1.0, hypothesis-3.69.2
collected 14 items                                                                                                                                  

tests/test_cssselect.py ..............                                                                                                        [100%]

============================================================= 14 passed in 0.51 seconds =============================================================
phase test: passed
```
I had some troubles with calling `pytest` directly in other packages, but here it seems to work as well. So I can change the script, but it would be nice to know why `setup.py` behaves so differently...

Reply:
Update (previously eaten by the GitHub interface): `setuptools-scm-py` also was a requirement, although without it I still got a different error than @danielj7. Easier to go with the direct `pytest` call here, anyway.